apps/codeforafrica/src/lib/data/blockify/ourWork.js

```diff
@@ -0,0 +1,18 @@
+import { getProjects } from "@/codeforafrica/lib/data/utils/projects";
+
+async function ourWork(block, api, context) {
+  const { query } = context;
+  const data = await getProjects(api, query);
+  const { docs: allProjects } = await api.getCollection("projects");
+  const projectTags = allProjects.map(({ tag }) => tag?.name).filter(Boolean);
```

Comment:
Are we just taking `tag.name`? I thought we would be using `tag.slag` for filtering.